### DISCUSSION TITLE
How to downgrade to OpenSSL 1.1.1

### DISCUSSION THREAD

**Discussion by yqi19** [openssl/openssl#23931] [Q&A]
^. My current openSSL version is OpenSSL 3.0.13 30 Jan 2024 (Library: OpenSSL 3.0.13 30 Jan 2024). I need OpenSSL 1.1.1 for experiments needed. How to downgrade openSSL 1.1.1. Thank you!

**Comment by kulkarniamit** [comment]
You can build it from [source](https://github.com/openssl/openssl/tree/OpenSSL_1_1_1-stable) and install in your custom test directory using `--prefix` option
```
See the appropriate file for build instructions:
        INSTALL         Linux, Unix, Windows, OpenVMS, ...
        NOTES.*         INSTALL addendums for different platforms
```
You can also refer [Compilation and Installation wiki](https://wiki.openssl.org/index.php/Compilation_and_Installation) to learn about configure options
#### NOTES
* Versions 1.1.1 and 1.0.2 are no longer supported. 
* https://www.openssl.org/blog/blog/2023/03/28/1.1.1-EOL/
* https://www.openssl.org/policies/releasestrat.html
